File: Go/factorial.go
package main

import "fmt"

func factorial(n int) int {
	if n < 2 {
		return 1
	}else{
		return n * factorial(n-1)
	}
}

func main() {
	fmt.Print("Enter a number: ")
	var n int
	fmt.Scanf("%d", &n)

	if n >= 0 {
		fmt.Println("Factorial de", n, "es", factorial(n))
	}else{
		fmt.Println("Choose another number")
	}
}

File: Python/factorial.py
def factorial(n):
    if(n == 1 or n == 0):
        return 1
    else:
        return n * factorial(n-1)
    

def main():
    
    n = int(input("Enter a number: "))
    
    if n >= 0:
        print(f"Factorial: {factorial(n)}")
    else:
        print(f"Choose another number")
  

if __name__ == "__main__":
    main()
